File: BOJ/4344.js
const input = require('fs').readFileSync('/dev/stdin').toString().split('\n').filter(Boolean);
for (let i = 1; i <= parseInt(input); i += 1) {
    const tc = input[i].toString().split(' ').map(el => parseInt(el));
    const avg = tc.slice(1).reduce((acc, cur) => acc + cur) / tc[0];
    let cnt = 0;

    tc.slice(1).forEach(el => el > avg ? cnt += 1 : 0);
    console.log(`${(cnt / tc[0] * 100).toFixed(3)}%`)
}

File: BOJ/4344.cpp
#include <stdio.h>

int main() {
    int tc, n[1001];
    scanf("%d", &tc);

    while (tc--) {
        int c = 0, avg = 0, sum = 0, cnt = 0;
        scanf("%d", &c);
        for (int i = 0; i < c; i++) {
            scanf("%d", &n[i]);
            sum += n[i];
        }
        avg = sum / c;
        for (int j = 0; j < c; j++) {
            if (n[j] > avg) cnt++;
        }
    printf("%.3f%%\n", (float)cnt / c * 100);
    }
}